tests/src/JIT/config/benchmark/project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dependencies": {
+    "Microsoft.DotNet.xunit.performance": "1.0.0-*",
+    "System.Console": "4.0.0-beta-*",
+    "System.Runtime": "4.0.20-beta-*",
+    "System.Runtime.Extensions": "4.0.10-beta-*",
+    "System.Numerics.Vectors": "4.1.1-beta-*",
```

Comment:
Oh I see you want to share this with all the benchmarks, including the SIMD ones we add. Looks good.